### ISSUE TITLE
npm version is out of sync with github version

### ISSUE THREAD

**Issue by GrosSacASac** [fingerprintjs/fingerprintjs2#463]
### Scenario

I use `npm i fingerprintjs2@latest`

I get an outdated version. It does not include the latest commits including this one https://github.com/Valve/fingerprintjs2/commit/9b631f52f48000370ad7b4e6861f5b32848ab032 for example

### Solution

Publish to npm

### Long term solution

We could do something like described here https://blog.npmjs.org/post/184553141742/easy-automatic-npm-publishes
So basically type `npm version major | minor | patch` will run tests, publish to npm, push to github, push tags for github (good for bower)

### temporary workaround

download from git with `npm i  Valve/fingerprintjs2`
workaround does not work until https://github.com/Valve/fingerprintjs2/issues/466 is fixed

**Comment by jonashaag** [comment]
Thanks! I published 2.1.0.

Please, remind me on a regular basis to cut a new release. I'm doing this in my spare time and forget to make releases every now and then. Maybe we can also find a permanent solution that doesn't rely on me or you remembering to make releases.

**Comment by GrosSacASac** [comment]
Could you publish the latest ?

**Comment by ccurtin** [comment]
@jonashaag 

Perhaps by using Github Actions, you can automate publishing to NPM when a new release is created?

https://medium.com/faun/automate-your-npm-publish-with-github-actions-dfe8059645dd

**Comment by Valve** [comment]
@markogresak please consider this as well in your PR as it's related to the work you're doing in https://github.com/Valve/fingerprintjs2/issues/483

**Comment by Valve** [comment]
a new version has been released, we'll add github actions soon

**Comment by Finesse** [comment]
The latest release is published to NPM. The publication process will be automated in v3 as described at https://github.com/fingerprintjs/fingerprintjs2/issues/466#issuecomment-689284650
